src/confcom/azext_confcom/security_policy.py

```diff
@@ -350,6 +350,7 @@ def _policy_serialization(self, pretty_print=False) -> str:
             return pretty_print_func(policy)
         return print_func(policy)
 
+    # pylint: disable=R0914, R0915
```

Comment:
Is this supposed to be here?

Reply:
yeah it violates too many statements and too many local variables but refactoring, in my opinion, would look worse